### PR TITLE
Tesseron Invoice

### PR DESCRIPTION
**Pull request by developmentLL**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Hello @developmentLL,

Thank you so much for your new submission with us.

As we ask all partner to register their new connectors, may I ask you please to register your connector by filling out our [registration form](https://aka.ms/ConnectorRegistration).

Please let me know if you have any questions.

Thank you so much.

File: certified-connectors/Tesseron Invoice/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,245 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tesseron ASM Ticket",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Tesseron Invoice/apiDefinition.swagger.json:4]: @developmentLL we have an existing connector with the same name "[Tesseron ASM Ticket](https://learn.microsoft.com/en-us/connectors/tesseronasmticket/)", can you please confirm if this is an update to the existing connector or is it a new connector? if it is new, then please change the title.

File: certified-connectors/Tesseron Invoice/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    "description": "This connector allows you to create, edit and search Tickets in your Tesseron ASM Instance.",
+    "contact": {
+      "name": "Tesseron ASM Support",
+      "email": "development@llnet.de",
+      "url": "https://www.docs.tesseron.de"
```
Comment (Amjed-Ayoub) [inline certified-connectors/Tesseron Invoice/apiDefinition.swagger.json:9]: @developmentLL please provide a valid URL.
image

File: certified-connectors/Tesseron Invoice/readme.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+## Prerequisites
+You will need the following to proceed:
+* A Tesseron Instance
+* A Tesseron licensed user
+* An API Key (Service: Invoice)
+
+
+## Supported Operations
+The connector supports the following operations:
+* `Create Inovice Position Note`: Create a position note with or without tickets
```
Comment (Amjed-Ayoub) [inline certified-connectors/Tesseron Invoice/readme.md:17]: @developmentLL all files should be free of grammatical errors and typos. here you have "Create Inovice...", can you please change to "Create Invoice...".

File: certified-connectors/Tesseron Invoice/readme.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+* `Create Activity Recording`: Add a new activity recordings to your profile
+
+
+## Obtaining Credentials
+Authentication is done via an API key. Please provide your API key and your instance URL als connection parameters.
+
+
+## Known Issues and Limitations
+* Sufficient user rights are mandatory to receive ticket information in specific areas
+* If the ticket creation is triggered multiple time at once there may be an incomplete ticket creation
```
Comment (Amjed-Ayoub) [inline certified-connectors/Tesseron Invoice/readme.md:27]: @developmentLL all files should be free of grammatical and syntax errors. here you have "...multiple time...", Maybe you meant to say "...multiple times...".

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/Tesseron Invoice/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+                "invoiceNoteId": {
+                  "type": "string",
+                  "description": "Invoice Note ID"
+                }
+              }
+            }
+          }
+        }
+      },
+      "summary": "Create Position Note",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Tesseron Invoice/apiDefinition.swagger.json:51]: @developmentLL the operation properties should be within the scope of post action. before or after the responses.
image.

Please fix all operations in the swagger file.

File: certified-connectors/Tesseron Invoice/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+                "ActivityRecordingId": {
+                  "type": "string",
+                  "description": "Activity Recording ID"
+                }
+              }
+            }
+          }
+        }
+      },
+      "summary": "Activity Recording ID",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Tesseron Invoice/apiDefinition.swagger.json:156]: @developmentLL the operation properties should be within the scope of post action. before or after the responses.

Please fix all operations in the swagger file.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by developmentLL** [comment]
New Pull Request will be made

**Pull request closed without merging**